src/Microsoft.Android.Sdk.ILLink/Microsoft.Android.Sdk.ILLink.csproj

```diff
@@ -1,17 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\Configuration.props" />
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <DefineConstants>NET5_LINKER</DefineConstants>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <OutputPath>$(XAInstallPrefix)xbuild\Xamarin\Android\</OutputPath>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.ILLink" Version="5.0.0-preview.3.20227.3" />
```

Comment:
Is it worth putting this version in `Configuration.props` so it is next to `$(DotNetPreviewVersion)`?

We should probably bump this number when bumping the build of .NET 6?

Reply:
Makes sense, updated.